### ISSUE TITLE
Running Maven TestRunner without Docker times out.

### ISSUE THREAD

**Issue by mattvv** [localstack/localstack#201]
When running the testRunner with maven, if I don't have docker running the test will fail.
Is there a way to get the stack running without docker or does the testrunner require docker?

Thanks

**Comment by whummer** [comment]
Docker should not be necessary to run tests from Java via `LocalstackTestRunner`. This seems to be related to https://github.com/localstack/localstack/issues/200 . Can you please try following the cleanup steps described on that other issue?

To confirm that the local installation is set up properly, can you please try running this command from a fresh clone of the repository:
```
make test-java
```

Please report back if that works, and post any relevant error output or stack traces. Thanks

**Comment by mattvv** [comment]
Thanks for the speedy reply @whummer 

I had to add 
```
        <dependency>
            <groupId>commons-io</groupId>
            <artifactId>commons-io</artifactId>
            <version>2.5</version>
        </dependency>
```
to the pomfile for the maven to actually build, before which I was getting a compilation failure. 
```[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project localstack-utils: Compilation failure: Compilation failure:
[ERROR] /Users/mvan/Projects/localstack/localstack/ext/java/src/main/java/cloud/localstack/LocalstackTestRunner.java:[16,28] error: package org.apache.commons.io does not exist
[ERROR] /Users/mvan/Projects/localstack/localstack/ext/java/src/main/java/cloud/localstack/LambdaExecutor.java:[17,28] error: package org.apache.commons.io does not exist
[ERROR] /Users/mvan/Projects/localstack/localstack/ext/java/src/main/java/cloud/localstack/LocalstackTestRunner.java:[140,4] error: cannot find symbol
[ERROR] class LocalstackTestRunner
[ERROR] /Users/mvan/Projects/localstack/localstack/ext/java/src/main/java/cloud/localstack/LambdaExecutor.java:[90,9] error: cannot find symbol
```

After this, not running docker I get
```
-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running cloud.localstack.BasicFunctionalityTest
Jul 25, 2017 10:38:58 AM cloud.localstack.LocalstackTestRunner setupInfrastructure
INFO: Waiting for infrastructure to be spun up
Jul 25, 2017 10:39:05 AM com.amazonaws.http.AmazonHttpClient createSocketFactoryRegistry
WARNING: SSL Certificate checking for endpoints has been explicitly disabled.
Jul 25, 2017 10:39:06 AM com.amazonaws.http.AmazonHttpClient createSocketFactoryRegistry
WARNING: SSL Certificate checking for endpoints has been explicitly disabled.
Jul 25, 2017 10:39:06 AM com.amazonaws.http.AmazonHttpClient createSocketFactoryRegistry
WARNING: SSL Certificate checking for endpoints has been explicitly disabled.
Tests run: 3, Failures: 0, Errors: 3, Skipped: 0, Time elapsed: 7.74 sec <<< FAILURE!
Running cloud.localstack.S3HttpsConnectionTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0 sec
Running cloud.localstack.SQSMessagingTest
Jul 25, 2017 10:39:06 AM com.amazonaws.http.AmazonHttpClient createSocketFactoryRegistry
WARNING: SSL Certificate checking for endpoints has been explicitly disabled.
Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0.557 sec <<< FAILURE!

Results :

Tests in error: 
  testLocalS3API(cloud.localstack.BasicFunctionalityTest): Unable to execute HTTP request: Connect to test.localhost.atlassian.io:4572 [test.localhost.atlassian.io/127.0.0.1] failed: Connection refused
  testLocalLambdaAPI(cloud.localstack.BasicFunctionalityTest): Unable to execute HTTP request: Connect to localhost:4574 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused
  testLocalKinesisAPI(cloud.localstack.BasicFunctionalityTest): Unable to execute HTTP request: Connect to localhost:4568 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused
  cloud.localstack.SQSMessagingTest: Unable to execute HTTP request: Connect to localhost:4576 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused
```

**Comment by whummer** [comment]
@mattvv The issue with commons-io should be fixed in the latest version `0.1.2`.

Regarding the other issue, can you please post some debug output:
```
echo $TMPDIR
ls -la $TMPDIR/localstack_install_dir
(cd $TMPDIR/localstack_install_dir; DEBUG=1 make infra)
```

Thanks

**Comment by whummer** [comment]
Closing as outdated / obsolete. Please open follow-up issues if the problem persists. Thanks
